v2/cf-nodejs-logging-support/src/lib/logger/source-utils.ts

```diff
@@ -38,7 +38,7 @@ export class SourceUtils {
         }
     }
 
-    getReqFieldValue(source: Source, record: any, req: any, res: any): string | undefined {
+    getReqFieldValue(source: Source, record: any, req: any, res: any, now?: Date): string | undefined {
```

Comment:
Can you rename now to sth. that indicates what this timestamp means? From my understanding this is the receive timestamp, right?

Reply:
I renamed "now" to "writtenAt"